iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/CatalogConstants.java

```diff
@@ -17,6 +17,8 @@ public final class CatalogConstants {
 
   static final String FEATURE_TOGGLE_STOP_CREATE = "stop_create";
 
+  public static final String CLIENT_TABLE_SCHEMA = "client.table.schema";
```

Comment:
it doesn't have to be public since it only gets referenced in the same package.
Even if you want to reuse it in client side, it's probably not a good idea to have a server-side constant used as a dependency on client side.